bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/size/vo/Library.java

```diff
@@ -0,0 +1,95 @@
+/*******************************************************************************
+ * Copyright 2025 Espressif Systems (Shanghai) PTE LTD. All rights reserved.
+ * Use is subject to license terms.
+ *******************************************************************************/
+package com.espressif.idf.ui.size.vo;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Main vo for holding all the data related to a memory object
+ * 
+ * @author Ali Azam Rana <ali.azamrana@espressif.com>
+ *
+ */
+public class Library
+{
```

Comment:
Please considering chaning the class name to more relevant - MemoryComponent, MemoryNode, etc